CesiumGltf/include/CesiumGltf/PropertyTableView.h

```diff
@@ -194,13 +195,44 @@ class PropertyTableView {
 
     PropertyType type = convertStringToPropertyType(pClassProperty->type);
     PropertyComponentType componentType = PropertyComponentType::None;
-    if (pClassProperty->componentType) {
+
+    // Handle enum component type indirection
+    if (type == PropertyType::Enum && pClassProperty->enumType) {
+      const auto& enumDefinitionIt =
+          this->_pEnumDefinitions->find(*pClassProperty->enumType);
+      if (enumDefinitionIt == this->_pEnumDefinitions->end()) {
+        callback(
+            propertyId,
+            PropertyTablePropertyView<uint8_t>(
+                PropertyTablePropertyViewStatus::ErrorInvalidEnum));
+        return;
+      } else {
+        componentType = convertStringToPropertyComponentType(
+            enumDefinitionIt->second.valueType);
+
+        if (componentType == PropertyComponentType::Float32 ||
+            componentType == PropertyComponentType::Float64) {
+          callback(
+              propertyId,
+              PropertyTablePropertyView<uint8_t>(
+                  PropertyTablePropertyViewStatus::ErrorInvalidEnum));
+          return;
+        }
+      }
```

Comment:
Might be nitpicky, but can be flatten the `if else` here so there's less indentation? It helps with readability at the end of the day

Reply:
I don't see how it can be flattened?

Reply:
Sorry, I just meant the outer if/else statement. Like this:

```suggestion
      if (enumDefinitionIt == this->_pEnumDefinitions->end()) {
        callback(
            propertyId,
            PropertyTablePropertyView<uint8_t>(
                PropertyTablePropertyViewStatus::ErrorInvalidEnum));
        return;
      }
        componentType = convertStringToPropertyComponentType(
            enumDefinitionIt->second.valueType);

        if (componentType == PropertyComponentType::Float32 ||
            componentType == PropertyComponentType::Float64) {
          callback(
              propertyId,
              PropertyTablePropertyView<uint8_t>(
                  PropertyTablePropertyViewStatus::ErrorInvalidEnum));
          return;
        }
      
```

Reply:
Same with all the others.

Reply:
@azrogers and this one.